### ISSUE TITLE
RFE - Teach Princess to use 'Careful Stand' (occasionally)

### ISSUE THREAD

**Issue by SimonLandmine** [MegaMek/megamek#2222]
Princess's behaviour has improved immeasurably over recent releases, which has highlighted one notable tendency - even when chances of successfully getting up are very low, Princess will repeatedly attempt to stand, often knocking out (or killing) the pilot as a result.
Would there be a way to set things so that if standing is above a certain difficulty, Princess would use Careful Stand (possibly also based on proximity of opponents)? Maybe also related to the fallShame or selfPreservation values?

**Comment by NickAragua** [comment]
She already uses careful stand. Just have to set the piloting caution to "pretty cautious".

**Comment by SimonLandmine** [comment]
Aha! Cheers! I'll feed that back to the folks who were querying.

**Comment by SimonLandmine** [comment]
It's been noted that Princess will continue attempting to stand when the target number is 13 or more ... would that be a situation to trigger Careful Stand, regardless of piloting caution? Or to eject if even Careful Stand won't cut it?

**Comment by gsparks3** [comment]
Often in a situation where the PSR is that high, Careful Stand is not possible (e,g. leg is destroyed, so MP is 1, which is too low to use Careful Stand - you need to have at least 3 MP available to get the PSR difficulty reduction). Similarly, a destroyed leg also prevents moving directly from Prone to Hull Down, which is another possible trick if MP is fine but the difficulty is high (e.g. a Locust with a bad pilot that skidded and fell over, but is otherwise undamaged), as that doesn't require a PSR and means you won't fall over if you fail further PSRs to stand up. 

In any case, it's a little odd to attempt to stand when the difficulty is high enough for it to be a guaranteed failure. 
![image](https://user-images.githubusercontent.com/59574316/96348124-5ef07600-1074-11eb-895d-b1f0f4cdd94b.png)

**Comment by SimonLandmine** [comment]
Fair point. Had a couple of Princess units faceplant themselves to death in my last battle.
But, where legs are both intact, but standing is difficult, might be worth increasing the frequency of 'careful stand'. Or, as you note, moving from Prone to Hull Down. (Does Princess even attempt that halfway step intentionally, other than by accident due to a only-just fall?)
